locales/en-US.yml

```
@@ -3090,6 +3092,7 @@ en-US:
       tv_episode_pages: On TV episode pages
       tv_season_pages: On TV season pages
     keyboard_shortcuts: Keyboard Shortcuts
+    login_to_change_preferences: Login to change keyboard shortcuts preferences under account settings
```

I had a chat with Travis about maintaining **TMDB's voice** in the app. He suggested a message that invites the user to login or sign up for an account in a more natural way.
```suggestion
    login_to_change_preferences: You can disable keyboard shortcuts by logging into your account.
```